structs.go

```diff
@@ -1336,57 +1336,168 @@ const (
+	ErrCodeMaximumNumberOfEmojisReached                            = 30008
+	ErrCodeTooManyReactions                                        = 30010
+	ErrCodeMaximumNumberOfGuildChannelsReached                     = 30013
+	ErrCodeMaximumNumberOfAttachmentsInAMessageReached             = 30015
+	ErrCodeMaximumNumberOfInvitesReached                           = 30016
+	ErrCodeMaximumNumberOfAnimatedEmojisReached                    = 30018
+	ErrCodeMaximumNumberOfSeverMembersReached                      = 30019
+	ErrCodeMaximumNumberOfGuildDiscoverySubcategoriesReached       = 30030
+	ErrCodeGuildAlreadyHasATemplate                                = 30031
+	ErrCodeMaximumNumberOfThreadParticipantsReached                = 30033
+	ErrCodeMaximumNumberOfBansForNonGuildMembersHaveBeenExceeded   = 30035
+	ErrCodeMaximumNumberOfBansFetchesHasBeenReached                = 30037
+	ErrCodeMaximumNumberOfUncompletedGuildScheduledEventsReached   = 30038
+	ErrCodeMaximumNumberOfStickersReached                          = 30039
+	ErrCodeMaximumNumberOfPruneRequestsHasBeenReached              = 30040
+	ErrCodeMaximumNumberOfGuildWidgetSettingsUpdatesHasBeenReached = 30042
+	ErrCodeMaximumNumberOfEditsToMessagesOlderThanOneHourReached   = 30046
+
+	ErrCodeUnauthorized                           = 40001
+	ErrCodeActionRequiredVerifiedAccount          = 40002
+	ErrCodeOpeningDirectMessagesTooFast           = 40003
+	ErrCodeSendMessagesHasBeenTemporarilyDisabled = 40004
+	ErrCodeRequestEntityTooLarge                  = 40005
+	ErrCodeFeatureTemporarilyDisabledServerSide   = 40006
+	ErrCodeUserIsBannedFromThisGuild              = 40007
+	ErrCodeTargetIsNotConnectedToVoice            = 40032
+	ErrCodeMessageAlreadyCrossposted              = 40033
+	ErrCodeAnApplicationWithThatNameAlreadyExists = 40041
+
+	ErrCodeMissingAccess                                                = 50001
+	ErrCodeInvalidAccountType                                           = 50002
+	ErrCodeCannotExecuteActionOnDMChannel                               = 50003
+	ErrCodeEmbedDisabled                                                = 50004
+	ErrCodeCannotEditFromAnotherUser                                    = 50005
+	ErrCodeCannotSendEmptyMessage                                       = 50006
+	ErrCodeCannotSendMessagesToThisUser                                 = 50007
+	ErrCodeCannotSendMessagesInVoiceChannel                             = 50008
+	ErrCodeChannelVerificationLevelTooHigh                              = 50009
+	ErrCodeOAuth2ApplicationDoesNotHaveBot                              = 50010
+	ErrCodeOAuth2ApplicationLimitReached                                = 50011
+	ErrCodeInvalidOAuthState                                            = 50012
+	ErrCodeMissingPermissions                                           = 50013
+	ErrCodeInvalidAuthenticationToken                                   = 50014
+	ErrCodeNoteTooLong                                                  = 50015
```

Comment:
I think it was added by mistake to the docs, bots shouldn't be able to access notes at all.